plugins/reporters/aosd/src/main/kotlin/Aosd21Reporter.kt

```diff
@@ -62,6 +62,7 @@ class Aosd21Reporter(override val descriptor: PluginDescriptor = Aosd21ReporterF
                         indexedPackages[it]?.index?.toLong()
                     },
                     components = indexedPackages.toComponents(input, project)
+                        .toSortedSet(compareBy { it.componentName })
```

Comment:
There was this larger effort to eliminate the use of sorted collections in favor of using annotation to do the sorting for the serialization only. I believe we should annotate the `components` property instead.

Reply:
Please note that the model still only uses a `Set`, not a `SortedSet`. I.e. users of the data model are not *forced* to use a sorted set, esp. when only using the data model in memory. And with my change even user that *want* to serialized are not forced to use a sorted set. In that sense my change is even less strict than enforcing sorting on any serialization.

Reply:
> Please note that the model still only uses a Set, not a SortedSet. 

Was aware of that.

> And with my change even users that want to serialize are not forced to use a sorted set.

I suppose with users your speaking of callers from within the same component. It's not obvious to me
whether there is need for using different orderings within the reporter.

...anyhow, I thought the annotation approach would be favorable by default, because it yields a more deterministic serialization.